### PR TITLE
en: add doc for migrate pd/tikv pod

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1403
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/maintain-a-kubernetes-node.md [hunk 1]
```diff
@@ -18,215 +18,460 @@ This document introduces how to perform a temporary or long-term maintenance tas
...
+
+## Reschedule PD Pods
+
+If a node will be offline for a long time, to minimize the impact on your application, you can reschedule the PD Pods on this node to other nodes in advance.
 
-## Maintain nodes that hold TiKV instances
+### If the node storage can be automatically migrated
 
-Migrating TiKV instances is relatively slow and might lead to unwanted data migration load on the cluster. For this reason, you need to choose different strategies as needed prior to maintaining the nodes that hold TiKV instances:
+If the node storage can be automatically migrated (such as EBS), to reschedule the PD Pod, you do not need to delete the PD member. You only need to transfer the PD Leader to another Pod and delete the old Pod.
```
Comment (qiancai) [inline en/maintain-a-kubernetes-node.md:181]:
```suggestion
If the node storage can be automatically migrated (such as EBS), to reschedule a PD Pod, you do not need to delete the PD member. You only need to transfer the PD Leader to another Pod and delete the old Pod.
```

File: en/maintain-a-kubernetes-node.md [hunk 2]
```diff
@@ -18,215 +18,460 @@ This document introduces how to perform a temporary or long-term maintenance tas
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
+
+### If the node storage cannot be automatically migrated
+
+If the node storage cannot be automatically migrated (such as local storage), to reschedule the PD Pod, you need to delete the PD member.
```
Comment (qiancai) [inline en/maintain-a-kubernetes-node.md:219]:
```suggestion
If the node storage cannot be automatically migrated (such as local storage), to reschedule a PD Pod, you need to delete the PD member.
```

File: en/maintain-a-kubernetes-node.md [hunk 3]
```diff
@@ -18,215 +18,460 @@ This document introduces how to perform a temporary or long-term maintenance tas
...
     ```
 
-    Make the TiKV instance offline:
+## Reschedule TiKV Pods
+
+If a node will be offline for a long time, to minimize the impact on your application, you can reschedule the TiKV Pods on this node to other nodes in advance.
+
+### If the node storage can be automatically migrated
+
+If the node storage can be automatically migrated (such as EBS), to reschedule the TiKV Pod, you do not need to delete the whole TiKV store. You only need to evict the TiKV Region Leader to another Pod and delete the old Pod.
```
Comment (qiancai) [inline en/maintain-a-kubernetes-node.md:295]:
```suggestion
If the node storage can be automatically migrated (such as EBS), to reschedule a TiKV Pod, you do not need to delete the whole TiKV store. You only need to evict the TiKV Region Leader to another Pod and delete the old Pod.
```

File: en/maintain-a-kubernetes-node.md [hunk 4]
```diff
@@ -18,215 +18,460 @@ This document introduces how to perform a temporary or long-term maintenance tas
...
     ```shell
-    watch pd-ctl -d store ${ID}
+    pd-ctl scheduler remove evict-leader-scheduler-${ID}
     ```
 
-5. Unbind the TiKV instance from the local drive of the node:
+### If the node storage cannot be automatically migrated
 
-    Get the `PesistentVolumeClaim` used by the Pod:
+If the node storage cannot be automatically migrated (such as local storage), to reschedule the TiKV Pod, you need to delete the whole TiKV store.
```
Comment (qiancai) [inline en/maintain-a-kubernetes-node.md:341]:
```suggestion
If the node storage cannot be automatically migrated (such as local storage), to reschedule a TiKV Pod, you need to delete the whole TiKV store.
```

File: en/maintain-a-kubernetes-node.md [hunk 5]
```diff
@@ -18,215 +18,460 @@ This document introduces how to perform a temporary or long-term maintenance tas
...
     ```
 
-6. Delete the TiKV instance:
+3. [Evict the TiKV Region Leader](#evict-tikv-region-leader) to another Pod.
+
+4. Take the TiKV Pod offline.
+
+    > **Note:**
+    >
+    > Before you take the TiKV Pod offline, make sure that the remaining TiKV Pods are not fewer than the TiKV replica number set in PD configuration (`max-replicas`, 3 by default). If there are not enough TiKV Pods, scale out TiKV Pods before you take the TiKV Pod offline.
```
Comment (qiancai) [inline en/maintain-a-kubernetes-node.md:365]:
```suggestion
    > Before you take the TiKV Pod offline, make sure that the remaining TiKV Pods are not fewer than the TiKV replica number set in PD configuration (`max-replicas`, 3 by default). If the remaining TiKV Pods are not enough, scale out TiKV Pods before you take the TiKV Pod offline.
```

**Review by qiancai — COMMENTED** [review]
rest LGTM

**Comment by ran-huang** [comment]
@qiancai PTAL again, thanks!

**Review by qiancai — APPROVED** [review]
LGTM

**Review by liubog2008 — APPROVED** [review]
LGTM

**Pull request merged**